### PR TITLE
Rpc v2

### PR DESCRIPTION
**Pull request by NinjaManatee**
Updated example to use RPC v2.

### CODE REVIEW

File: Finsemble/FinsembleBridge.cs [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+
+			/// <summary>
+			/// Property name for endpoint to which the data is being sent.
+			/// </summary>
+			public static readonly string Endpoint = "endpoint";
+
+			/// <summary>
+			/// Value indicating that the method has a callback.
+			/// </summary>
+			public static readonly string HasCallbackString = "**cb**";
```
Comment (thorsent) [inline Finsemble/FinsembleBridge.cs]: @NinjaManatee Should be three asterisks on either side: ***cb***

File: FinsembleRPCLinkerExample/LinkerRPCExample.cs [hunk 2]
```diff
@@ -110,10 +111,29 @@ private void PublishButton_Click(object sender, EventArgs e)
...
+
+			// Create arguments array.
+			var args = new JArray()
+			{
+				parameters
+			};
 
 			// Publish the data to the Finsemble Linker using the data type.
-			bridge.SendRPCCommand(Topic.Publish, new JObject { ["dataType"] = currentDataType, ["data"] = data });
+			FSBL.SendRPCMessage(
```
Comment (thorsent) [inline FinsembleRPCLinkerExample/LinkerRPCExample.cs]: @NinjaManatee can we change to FSBL.RPC() in order to have the same interface as Sidd's vesion?

**Comment by NinjaManatee** [comment]
@thorsent I just saw this change request. I've made it and it should be good to merge now.

**Comment by thorsent** [comment]
@NinjaManatee Can you go ahead and merge? I'm not set up to QA test java and .net stuff so I prefer someone else merge, in case I break something and can't fix.

**Pull request merged**